NewForce-Cohort-9/newforce-c9-tabloid-cli-TabloidCLI
# Tyler branch1

# Description

Added all base functionality as it relates to posts.  Capabilities for adding, deleting, viewing, and editing post should be functioning.

Fixes # (issue)

## Type of change



# Testing Instructions

-Go to tylerBranch1
-Run app
-Choose option 4 for post management
-Choose option 1 to list posts (there should be one to begin) and verify they show
-Choose option 3 to create a post.  Create a post.  Give it whatever name and URL you like.  If you try to make a post with no author or blog, further instructions should appear.
-Choose option 4 to edit your post.  You should be able to edit any or none of the options.  When editing the ID's (author and blog), a list of options to choose from should appear.
-Choose option 5 to delete your post.

-At any juncture, you should be able to list your posts and see your relevant additions or changes appear.



# Checklist:
